Module_Postman/activities/Act7_API_Documentation_in_Postman/7.md

```diff
@@ -0,0 +1,19 @@
+<!--title={Publishing Our Documentation}-->
+
+<!--badges={Web Development:}-->
+
+<!--conecpts={}-->
+
+Now that we have finally finished creating our API documentation, we can head on over to the postman website and check and see what our it looks like. To do so, simply click on the arrow button next to our collection name and click the view in web button.
```

Comment:
Just a headsup that we recently released the ability to edit docs in the browser too..